lib/howl/ui/menu_popup.moon

```diff
@@ -46,6 +46,9 @@ class MenuPopup extends Popup
     page_up: => @list\prev_page!
 
     on_unhandled: (event, source, translations, self) ->
+      -- if a bare modifier such as just 'ctrl', don't close popup
+      return if bindings.is_modifier translations
+      -- any other not character such as home or escape closes the popup
```

Comment:
nit: would `non-character` be better here?

Reply:
`escape` is a non-character but that should close the popup (also `home`, `end`). Also we already have `.character` which is kind of the inverse of non-character. We really want to identify the modifiers like ctrl, shift, alt only.